public/app/features/alerting/unified/rule-list/RuleListPageTitle.tsx

```diff
@@ -21,17 +22,17 @@ export function RuleListPageTitle({ title }: { title: string }) {
     window.location.reload();
   };
 
-  const buttonConfig: ButtonProps & { text: string; 'data-testid': string } = listViewV2Enabled
+  const { text, ...configToUse }: ButtonProps & { text: string; 'data-testid': string } = listViewV2Enabled
     ? {
         variant: 'secondary',
         icon: undefined,
-        text: 'Go back to the old look',
+        text: t('alerting.rule-list.toggle.go-back-to-old-look', 'Go back to the old look'),
```

Comment:
Translation eslint rule doesn't catch this kind of thing (yet), and is best to translate it even if it's experimental functionality